app/screens/Reusable/Acknowledgement/AcknowledgementTitleSubtitle/AcknowledgementTitleSubtitle.tsx

```diff
@@ -67,19 +70,24 @@ export const AcknowledgementTitleSubtitleScreen: React.FC<IAcknowledgementTitleS
       if (isTitleHaveNickname) {
         return (
           <View style={generalStyles.row}>
-            {_.map(titleInfo?.textArray, (item: string) => {
-              if (item === titleInfo?.nicknameType) {
-                return <ExtendedText>{nickname}</ExtendedText>;
-              } else {
-                return <ExtendedText>{t(item)}</ExtendedText>;
+            {_.map(titleArray, (item: string) => {
+              switch (item) {
+                case NicknameType.Parent:
+                  // TODO: change to parentNickname when state will be update
+                  return <ExtendedText>{DUMMY_PARENT_NICKNAME}</ExtendedText>;
+                case NicknameType.Child:
+                  // TODO: change to childNickname when state will be update
+                  return <ExtendedText>{DUMMY_CHILD_NICKNAME}</ExtendedText>;
+                default:
+                  return <ExtendedText>{t(item)}</ExtendedText>;
```

Comment:
we render the same elements? we can just select variable, I think all of them will have the same styling